src/components/modules/puzzles/puzzlesBanner/PuzzlesBanner.tsx

```diff
@@ -1,11 +1,18 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { CacheProvider } from '@emotion/core';
 import createCache from '@emotion/cache';
 import { Container } from '@guardian/src-layout';
 import { Button } from '@guardian/src-button';
 import { SvgArrowDownStraight, SvgArrowUpStraight } from '@guardian/src-icons';
+import {
+    createClickEventFromTracking,
+    createViewEvenetFromTracking,
```

Comment:
Just a small typo in this `createViewEvenetFromTracking` name

Reply:
Thanks!